commit bbb3d2a1e2f8799e2b8535002e8edf3f9416c145
Author: AnatomicJC <jcvassort@mandriva.com>
Date:   Thu Jul 10 16:36:48 2014 +0200

    pulse-update-manager: Add Univention support
    
    Univention is based on Debian

diff --git a/pulse2/services/pulse2/pulse_update_manager/src/pulse-update-manager.py b/pulse2/services/pulse2/pulse_update_manager/src/pulse-update-manager.py
index ff6ac4e..c3fb1e5 100755
--- a/pulse2/services/pulse2/pulse_update_manager/src/pulse-update-manager.py
+++ b/pulse2/services/pulse2/pulse_update_manager/src/pulse-update-manager.py
@@ -30,12 +30,15 @@ if __name__ == '__main__':
     platform = platform().lower()
     
     # Init updateHandler according to system platform
-    if 'debian' in platform:
+    if 'debian' in platform or 'univention' in platform:
         from debianHandler import debianUpdateHandler
         updateHandler = debianUpdateHandler(platform)
     elif 'window' in platform:
         from windowsHandler import windowsUpdateHandler
         updateHandler = windowsUpdateHandler(platform)
+    else:
+        print 'Unsupported operating system'
+        sys.exit(1)
     
     # Disabling native update service
     # ugly try, except, but not really important to continue
